src/Raven.Server/Documents/Includes/IncludeTimeSeriesCommand.cs

```diff
@@ -1,29 +1,37 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Linq;
+using Raven.Client;
 using Raven.Client.Documents.Operations.TimeSeries;
 using Raven.Server.Documents.Handlers;
 using Raven.Server.ServerWide.Context;
+using Sparrow;
 
 namespace Raven.Server.Documents.Includes
 {
     public class IncludeTimeSeriesCommand
     {
         private readonly DocumentsOperationContext _context;
-        private readonly Dictionary<string, HashSet<TimeSeriesRange>> _timeSeriesRangesBySourcePath;
+        private readonly Dictionary<string, HashSet<AbstractTimeSeriesRange>> _timeSeriesRangesBySourcePath;
+        private readonly Dictionary<string, Dictionary<string, (long Count, DateTime Start, DateTime End)>> _timeSeriesStatsPerDocumentId;
 
         public Dictionary<string, Dictionary<string, List<TimeSeriesRangeResult>>> Results { get; }
 
-        public IncludeTimeSeriesCommand(DocumentsOperationContext context, Dictionary<string, HashSet<TimeSeriesRange>> timeSeriesRangesBySourcePath)
+        public IncludeTimeSeriesCommand(DocumentsOperationContext context, Dictionary<string, HashSet<AbstractTimeSeriesRange>> timeSeriesRangesBySourcePath)
         {
             _context = context;
             _timeSeriesRangesBySourcePath = timeSeriesRangesBySourcePath;
 
+            _timeSeriesStatsPerDocumentId = new Dictionary<string, Dictionary<string, (long Count, DateTime Start, DateTime End)>>(StringComparer.OrdinalIgnoreCase);
             Results = new Dictionary<string, Dictionary<string, List<TimeSeriesRangeResult>>>(StringComparer.OrdinalIgnoreCase);
         }
 
         public void Fill(Document document)
         {
+            if (document == null)
```

Comment:
When can it be `null` ?

Reply:
if requested doc id wasnt found in the storage